src/sentry/features/__init__.py

```diff
@@ -132,7 +132,6 @@
 default_manager.add("organizations:performance-issues-all-events-tab", OrganizationFeature, True)
 default_manager.add("organizations:performance-onboarding-checklist", OrganizationFeature, True)
 default_manager.add("organizations:performance-span-histogram-view", OrganizationFeature, True)
-default_manager.add("organizations:performance-suspect-spans-view", OrganizationFeature, True)
```

Comment:
In general to do this cleanly every time it should be essentially the reverse of adding a flag, you should consider that each step should be forward/backwards compatible, this should be safe even if the flag isn't fully rolled out by mistake:
- backend "restricted" conditions removed first so no 404s are thrown for either side of the flag
- frontend
- Flag / feature last. The frontend might depend on the flag existing (being true), and you might want to remove the 404s first.